### PR TITLE
xtables-addons: update to 3.9, fix build on kernel 5.4

### PR DESCRIPTION
**Pull request by dengqf6**
Maintainer: @jow-
Compile tested: ramips
Run tested: ramips

Description:
Update xtables-addons to 3.9
Fix packetscript and rtsp build on kernel 5.4

### CODE REVIEW

File: net/xtables-addons/patches/002-restore-support-for-Linux-4.14.patch [hunk 1]
```diff
@@ -34,11 +30,9 @@ index d08354a..faf5dd8 100644
  #endif
  
  #if defined(CONFIG_NF_CONNTRACK) || defined(CONFIG_NF_CONNTRACK_MODULE)
-diff --git a/extensions/pknock/xt_pknock.c b/extensions/pknock/xt_pknock.c
-index c76901a..7be0345 100644
 --- a/extensions/pknock/xt_pknock.c
 +++ b/extensions/pknock/xt_pknock.c
-@@ -357,10 +357,18 @@ has_logged_during_this_minute(const struct peer *peer)
+@@ -356,10 +356,18 @@ has_logged_during_this_minute(const stru
```
Comment (neheb) [inline net/xtables-addons/patches/002-restore-support-for-Linux-4.14.patch]: This has always struck me as amusing.

Do you know what is responsible for this?

Comment (dengqf6) [reply net/xtables-addons/patches/002-restore-support-for-Linux-4.14.patch]: What do you mean?

Comment (neheb) [reply net/xtables-addons/patches/002-restore-support-for-Linux-4.14.patch]: git diff creates those long lines. GNU diff (or whatever it it) shortens them. No idea why though.

Comment (ldir-EDB0) [reply net/xtables-addons/patches/002-restore-support-for-Linux-4.14.patch]: It is the 'quilt refresh' process that does the reduction in context, the stripping of index and stripping of comments from the non-topmost patch in a patch file containing multiple patches.

It is advisable to run 'make package/foo/refresh' when updating packages/patches to reduce the level of patch format churn and catch any fuzz.

Ensuring .quiltrc is setup correctly is a good idea too - https://openwrt.org/docs/guide-developer/build-system/use-patches-with-buildsystem

**Pull request merged**